packages/ui/src/utils/TypesUtil.ts

```diff
@@ -181,3 +181,27 @@ export type TagType = 'main' | 'shade'
 export type AccountEntryType = 'icon' | 'image'
 
 export type ThemeType = 'dark' | 'light'
+
+export type InputType =
```

Comment:
Maybe we can re-use default typescript types for `HtmlInputElement` here?
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.htmlinputelement.html